src/boot/efi/util.h

```diff
@@ -194,9 +191,9 @@ void hexdump(const char16_t *prefix, const void *data, size_t size);
         }
 
 #if defined(__i386__) || defined(__x86_64__)
-void beep(UINTN beep_count);
+void beep(size_t beep_count);
 #else
-static inline void beep(UINTN beep_count) {}
+static inline void beep(size_t beep_count) {}
```

Comment:
this one sounds strange, no? a numberof beeps is not a size of memory objects. might be a good candidate for just `unsigned`?

Reply:
We use size_t for counting things too, but whatever. :roll_eyes: